pkg/target/alertmanager/alertmanager.go

```diff
@@ -0,0 +1,145 @@
+package alertmanager
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/kyverno/policy-reporter/pkg/crd/api/policyreport/v1alpha2"
+	"github.com/kyverno/policy-reporter/pkg/target"
+	targethttp "github.com/kyverno/policy-reporter/pkg/target/http"
+)
+
+// Alert represents an AlertManager alert
+type Alert struct {
+	Labels      map[string]string `json:"labels"`
+	Annotations map[string]string `json:"annotations"`
+	StartsAt    time.Time         `json:"startsAt"`
+	EndsAt      time.Time         `json:"endsAt"`
+}
+
+// Options to configure the AlertManager target
+type Options struct {
+	target.ClientOptions
+	Host         string
+	Headers      map[string]string
+	CustomFields map[string]string
+	HTTPClient   targethttp.Client
+}
+
+type client struct {
+	target.BaseClient
+	host         string
+	headers      map[string]string
+	customFields map[string]string
+	client       targethttp.Client
+	logger       *zap.Logger
```

Comment:
would prefer to use zap.L() directly, like its done in the other target implementations